### ISSUE TITLE
Can't change Netlify identity URL given on first run

### ISSUE THREAD

**Issue by mattgif** [netlify/netlify-cms#1704]
Running Netlify CMS with Gatsby, I went to `http://localhost:9000/admin/#/` and clicked "Login with Netlify Identity." I pasted my site url at the prompt, including the `http://` portion. Now the CMS is trying to hit `https://http://my-site-url.netlify.com/.netlify/identity` and (of course) failing. 

I can't find a way to change this. After rebuilding and serving the site, I keep getting the same 'Failed to load' modal, with a 'forgot password' button. 

How can I point the CMS to the right URL?

**Comment by mattgif** [comment]
Got it, thanks!

**Comment by renschler** [comment]
@tech4him1 I'm not sure if I should file a separate issue for this, but I keep getting the "Failed to load settings from https://mywebsite.com/.netlify/identity" when I try to use the widget during local development. Is there something I'm doing wrong?

**Comment by tomrutgers** [comment]
@renschler Did you enable Netlify Identity and Git Gateway?

**Comment by renschler** [comment]
I don't have git gateway enabled (afaik), but this is just for the gatsby-hello-world example, i'm not using netlify-cms (so maybe this thread is misplaced - i apologize). 

I do have netlify identity enabled, and it seems to work fine in production, it just doesn't work locally.

**Comment by renschler** [comment]
Update: I develop in Brave Browser, and I had shields down, but things weren't working. They did work when I visited the production site though (even with shields up).

When I tested in Chrome things worked fine for local dev.

So I'm not sure what is causing the issue with Brave local development but this is the error in my console:
`localhost/:1 Access to fetch at 'https://smartscalesync.com/.netlify/identity/settings' from origin 'http://localhost:8000' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: Redirect is not allowed for a preflight request.`

**Comment by renschler** [comment]
ok no worries thanks

**Comment by yo-it-engineer** [comment]
> Currently, you need to run `localStorage.removeItem("netlifySiteURL")` in the console to reset the URL, and then reload the page. This is an upstream issue that is in the process of being fixed: [netlify/netlify-identity-widget#148](https://github.com/netlify/netlify-identity-widget/issues/148).

I'm sorry, but can anyone provide more step on how to run the `localStorage.removeItem("netlifySiteURL")`  in the console? 

I started up netlify-cms locally and entered the wrong Netlify identify URL and now I'm stuck on the error page.

After some googling, found out what you meant by localStorage(of the browser) and (developer)console(of the browser)...:
https://github.com/netlify/netlify-identity-widget/pull/70/files

**Comment by theredwillow** [comment]
I can modify the localStorage, but I can't get the repository URL to change under Identity > Git Gateway. I still need help.
